components/editor/TextEditor.tsx

```diff
@@ -1,15 +1,22 @@
 'use client';
 
+import { DragEventHandler } from 'react';
 import { Editor, EditorContent } from '@tiptap/react';
 import styled from 'styled-components';
 
-const TextEditor = ({ editor }: { editor: Editor | null }) => {
-  return (
+interface editorProps {
+  editor: Editor | null;
+  handleDragOver: DragEventHandler<HTMLDivElement> | undefined;
+  handleDrop: DragEventHandler<HTMLDivElement> | undefined;
+}
+
+const TextEditor = ({ editor, handleDragOver, handleDrop }: editorProps) => (
```

Comment:
사실 이거 컨벤션 대로면...
```suggestion
const TextEditor = (props: editorProps) => (
  const { editor, handleDragOver, handleDrop } = props;
```
요래 해주는게 맞아요!